test/test_util/proptypes_tester.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+library test_util.prop_utils;
+
+import 'package:over_react_test/over_react_test.dart';
+import 'package:test/test.dart';
+
+void testPropTypes(dynamic instanceBuilder()) {
```

Comment:
Was this meant to be committed? It's unused and seems unfinished

Reply:
yeah, ill remove this, it was my starting to think about how to make a testing utility but we have a seperate ticket for that now.